tests/helpers/test_condition.py

```diff
@@ -40,12 +41,14 @@ def calls(hass: HomeAssistant) -> list[ServiceCall]:
 
 
 @pytest.fixture(autouse=True)
-def setup_comp(hass: HomeAssistant) -> None:
+async def setup_sun(hass: HomeAssistant) -> Generator[None, None, None]:
```

Comment:
Note: if I remove the fixture entirely, then the tests still pass and there is no issue with lingering timers, so I am not quite sure what the fixture does here...

To be on the safe side, I have migrated to use a config-entry with load/unload.